src/contribute/contribution-guide.md

```diff
@@ -58,30 +58,30 @@ If interested, you can also join the other [mailing lists]({{ site.baseurl
 We use the Apache Software Foundation's
 [JIRA](https://issues.apache.org/jira/browse/BEAM) as an issue tracking and
 project management tool, as well as a way to communicate among a very diverse
-and distributed set of contributors. To be able to gather feedback, avoid
-frustration, and avoid duplicated efforts all Beam-related work should be
-tracked there.
+and distributed set of contributors. To avoid duplicated efforts,
+significant Beam-related work should be tracked there.
 
 If you do not already have an Apache JIRA account, sign up
 [here](https://issues.apache.org/jira/).
 
 If a quick
 [search](https://issues.apache.org/jira/issues/?jql=project%3DBEAM%20AND%20text%20~%20%22the%20thing%20I%20want%20to%20contribute%22)
 doesn’t turn up an existing JIRA issue for the work you want to contribute,
-create it. Please discuss your idea with a committer or the [component
+feel free to create one. Please discuss your idea with a committer or the [component
 lead](https://issues.apache.org/jira/browse/BEAM/?selectedTab=com.atlassian.jira.jira-projects-plugin:components-panel)
 in JIRA or, alternatively, on the developer mailing list.
 
 If there’s an existing JIRA issue for your intended contribution, please
-comment about your intended work. Once the work is understood, a committer will
+comment about your intended work. A committer can
```

Comment:
I think you can drop this paragraph except the "you ned the 'contributor' role to be assigned a JIRA".

Reply:
I'm going to leave it for now, but this should not be the last pass over this guide :) .